README.md

```diff
@@ -11,17 +11,27 @@ Course bot for performing laboratory works for "Computer Architecture" disciplin
 - `/start` -- зарегистрироваться, укажите своё имя как в ведомости и свою группу.
 - `/whoami` -- посмотреть информацию в реестре о себе (имя, группа, ID).
 
-### Для лабораторной работы №1:
+Первым делом вы должны запустить команду `/start` и ввести свои данные. Бот выведет `\whoami` для проверки. В случае ошибки в имени вы можете изменить его с помощью `/renameme` (эквивалента для группы пока нет).
```

Comment:
Only / in commands. If I don't mistaken.

Reply:
Fixed.